.eslintrc

```diff
@@ -1,16 +1,7 @@
 {
-  "parser": "babel-eslint",
-  "extends": ["airbnb", "prettier"],
-  "plugins": ["prettier"],
-  "rules": {
-    "linebreak-style": 0,
-    "react/jsx-filename-extension": 0,
-    "react/destructuring-assignment": 0,
-    "import/no-extraneous-dependencies": 0,
-    "prettier/prettier": "error"
-  },
+  "extends": ["react-app", "plugin:prettier/recommended", "prettier/react"],
```

Comment:
The `plugin:prettier/recommended` config adds the plugin, extends the `eslint-config-prettier` and adds the `prettier/prettier` error rule.